preliminary-work/nematic_order.py

```diff
@@ -0,0 +1,41 @@
+import flowermd
+import hoomd
+import gsd
+import numpy as np
+import freud
+import time
+
+
+def nematic_order(gsd_file):
```

Comment:
Let's add a parameter here for the directory (i.e. reference vector) that is used.